services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.tsx

```diff
@@ -402,6 +409,42 @@ export const RateDetails = ({
                                         onFileItemsUpdate={onFileItemsUpdate}
                                     />
                                 </FormGroup>
+                                <FormGroup
+                                    error={showFieldErrors(
+                                        errors.rateProgramIDs
+                                    )}
+                                >
+                                    <Label htmlFor="programIDs">
+                                        Programs this rate certification covers
+                                    </Label>
+                                    {showFieldErrors(errors.rateProgramIDs) && (
+                                        <PoliteErrorMessage>
+                                            {errors.rateProgramIDs}
+                                        </PoliteErrorMessage>
+                                    )}
+                                    <Field name="programIDs">
+                                        {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
```

Comment:
have we used this syntax elsewhere? Wrapping a component in `<Field>` is new to me

Reply:
This was used in SubmissionType, I replaced the `Select` component with our `ProgramSelect` essentially the same.